tools/crm_mon.c

```diff
@@ -87,6 +87,7 @@ static pcmk__supported_format_t formats[] = {
     CRM_MON_SUPPORTED_FORMAT_CURSES,
 #endif
     PCMK__SUPPORTED_FORMAT_HTML,
+    PCMK__SUPPORTED_FORMAT_LOG,
```

Comment:
Drop this line, we won't expose the log formatter to end users

Reply:
Done.

> This is perfect, with one trivial tweak. I tested it on a few of the scheduler test inputs and I think it looks great.

Thanks)